### ISSUE TITLE
allow renaming token title

### ISSUE THREAD

**Issue by baptx** [jwhitbeck/otpn900#2]
It would be nice to be able to rename a token title through the user interface. There is probably a workaround by editing the configuration file but I have not searched where it is located yet.

**Comment by jwhitbeck** [comment]
Wow I didn't realize that some people were still using this project :)

I unfortunately no longer have a Maemo device or SDK to iterate on the code. I would be happy to accept a PR or hand-off this project to a new maintainer.

**Comment by baptx** [comment]
I bought my first N900 last year :D (since Neo900 project was cancelled...)
I don't use it as my main device but your 2 factor authentication app can be useful in case my main device (Jolla smartphone) stops working.

**Comment by sicelo** [comment]
> Wow I didn't realize that some people were still using this project :)
> 

I have been using it ever since my comment on issue #1 

Now that there's also [Maemo Leste](https://github.com/maemo-leste/), you may be interested to know that I was able to build otpn900 there, and it will soon be available in Leste's repos, https://github.com/maemo-leste-extras/otpn900

This is it running on a Motorola Droid 4
![droid4-otp](https://user-images.githubusercontent.com/3158319/73962038-5c9b2100-4916-11ea-84df-b8f3bf10d188.png)

**Comment by sicelo** [comment]
> It would be nice to be able to rename a token title through the user interface. There is probably a workaround by editing the configuration file but I have not searched where it is located yet.

I agree doing it via UI would be best. In the meantime, the config file is at `/home/user/.config/otp/tokens`

**Comment by jwhitbeck** [comment]
> I have been using it ever since my comment on issue #1
> 
> Now that there's also [Maemo Leste](https://github.com/maemo-leste/), you may be interested to know that I was able to build otpn900 there, and it will soon be available in Leste's repos, https://github.com/maemo-leste-extras/otpn900

Well that made my day. Good luck with Maemo Leste!

**Comment by baptx** [comment]
@sicelo I noticed another issue. If we want to change the display order of tokens, we cannot do it from the user interface either and also have to edit `/home/user/.config/otp/tokens`.

**Comment by sicelo** [comment]
someday i hope to add that :smile: 

on Maemo Leste side, we've recently made some minor improvements that fix https://github.com/jwhitbeck/otpn900/issues/1 .. see https://github.com/maemo-leste-extras/maeotp/commits/master/

@baptx you're still using it with Fremantle? I don't have a build environment for that, but if you do, it should be possible to back port the improvements from Leste

**Comment by baptx** [comment]
@sicelo Yes but I don't use the N900 often, I just keep it nearby in case my main device fails.
